packages/date-picker/src/_subcomponents/DateInput/DateInput.module.scss

```
@@ -1,3 +1,9 @@
+@import "~@kaizen/design-tokens/sass/spacing";
+
 .dateInput {
   width: 100%;
+
+  .dateInputInput {
+    margin-top: $spacing-xs;
+  }
 }
```

Apologies I must not have been very clear with my ask! I meant remove the nesting altogether, because CSS Modules will ensure it doesn't conflict with any other classes of the same name in another file.

Also, I missed it earlier, but we should be using `$spacing-{pxValue}` (in this case `$spacing-6`) moving forward, as the t-shirt sizes are deprecated legacy tokens.

```suggestion
}

.input {
  margin-top: $spacing-6;
}
```